### ISSUE TITLE
EOF errors in content creator webviews -  if `--overwrite` is needed 

### ISSUE THREAD

**Issue by alisonlhart** [ansible/vscode-ansible#1730]
### Summary

When using the Create Ansible Playbook Project or Create Ansible Collection Project webviews, an EOL error appears if there are files that will be overwritten. 

### Extension version

Ansible extension version 24.12.1

### VS Code version

1.95.3

### Ansible Version

```shell
$ ansible --version
ansible [core 2.16.11]
  config file = /home/ali/code/vscode-extension/vscode-ansible/examples/ansible.cfg
  configured module search path = ['/home/ali/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.12/site-packages/ansible
  ansible collection location = /home/ali/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.12.6 (main, Sep  9 2024, 00:00:00) [GCC 13.3.1 20240522 (Red Hat 13.3.1-1)] (/usr/bin/python3)
  jinja version = 3.1.4
  libyaml = True
```

### OS / Environment

Fedora 40
ansible-creator 24.12.0

### Relevant log output

```shell
------------------------------------ ansible-creator logs ------------------------------------
 Warning: /home/ali/.ansible/collections/ansible_collections/test/test/galaxy.ym
          l already exists
 Warning: /home/ali/.ansible/collections/ansible_collections/test/test/plugins/f
          ilter/hello_world.py already exists
 Warning: /home/ali/.ansible/collections/ansible_collections/test/test/.devconta
          iner/devcontainer.json already exists
 Warning: /home/ali/.ansible/collections/ansible_collections/test/test/.devconta
          iner/docker/devcontainer.json already exists
 Warning: /home/ali/.ansible/collections/ansible_collections/test/test/devfile.y
          aml already exists
[97mFiles in the destination directory will be overwritten. Do you want to proceed? (y/n)[0m: Command failed: ansible-creator init collection test.test /home/ali/.ansible/collections/ansible_collections --no-ansi 
Traceback (most recent call last):
  File "/home/ali/.local/bin/ansible-creator", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/ali/.local/lib/python3.12/site-packages/ansible_creator/cli.py", line 95, in main
    cli.run()
  File "/home/ali/.local/lib/python3.12/site-packages/ansible_creator/cli.py", line 82, in run
    subcommand(config=Config(**self.args, output=self.output)).run()
  File "/home/ali/.local/lib/python3.12/site-packages/ansible_creator/subcommands/init.py", line 65, in run
    self._scaffold()
  File "/home/ali/.local/lib/python3.12/site-packages/ansible_creator/subcommands/init.py", line 164, in _scaffold
    answer = ask_yes_no(question)
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/ali/.local/lib/python3.12/site-packages/ansible_creator/utils.py", line 394, in ask_yes_no
    answer = input(f"{Color.BRIGHT_WHITE}{question} (y/n){Color.END}: ").lower()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
EOFError: EOF when reading a line
```

**Comment by abhikdps** [comment]
Won't this be resolved once we fix the ansible-creator bug reported [here](https://github.com/ansible/ansible-creator/issues/343) and then we start adding the `--no-overwrite` flag with the creator commands used in the webview?

**Comment by alisonlhart** [comment]
This is resolved as of ansible-creator 24.12.1 release! https://github.com/ansible/ansible-creator/releases/tag/v24.12.1
